notebooks/05_Kubeflow_Pipeline/05_04_Pipeline_SageMaker.ipynb

```diff
@@ -84,7 +84,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> Typically in a production environment, you would assign fine-grained permissions depending on the nature of actions you take and leverage tools like [IAM Role for Service Account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) for securing access to AWS resources but for simplicity we will assign AmazonSageMakerFullAccess IAM policy.  You can read more about granular policies [here](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html) \n",
+    "> Typically in a production environment, you would assign fine-grained permissions depending on the nature of actions you take and leverage tools like [IAM Role for Service Account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) for securing access to AWS resources but for simplicity we will assign AmazonSageMakerFullAccess and AmazonS3FullAccess IAM policy.  You can read more about granular policies [here](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html) \n",
```

Comment:
Can you update following texts as well?

like 

```
create Kubernetes secrets aws-secret with Sagemaker policies. Please make sure to create aws-secret in kubeflow namespace.
```

should be Sagemaker and S3.

Reply:
```
b) create an IAM execution role for Sagemaker so that the job can assume this role in order to perform Sagemaker actions. Make a note of this role as you will need it during pipeline creation step
```

Reply:
These were fixed in the latest version.